tasks/main.yml

```diff
@@ -1,235 +1,58 @@
 ---
-
 - name: Set version specific variables
   include_vars: "{{ item }}"
   with_first_found:
-    - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+    - "{{ ansible_distribution }}_{{ ansible_distribution_version }}.yml"
+    - "{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.yml"
     - "{{ ansible_distribution }}.yml"
     - "{{ ansible_os_family }}.yml"
     - "default.yml"
 
 - name: Check if only NTP is needed
   set_fact:
-    timesync_mode: 1
+    timesync_mode: ntp
   when: timesync_ptp_domains|length == 0
 
 - name: Check if single PTP is needed
   set_fact:
-    timesync_mode: 2
+    timesync_mode: ptp
   when:
     - timesync_mode is not defined
-    - timesync_ntp_servers|length == 0
-    - timesync_ptp_domains|length == 1
+    - timesync_ntp_servers|length in [0,1]
```

Comment:
is this correct? it is not equivalent to the previous version.

Reply:
my bad.